src/main/java/io/pravega/connectors/presto/decoder/JsonRowDecoderFactory.java

```diff
@@ -12,8 +10,11 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *
+ * Note: This class file is from PrestoDb
```

Comment:
Minor: can you add the commit id in PrestoDB from which you extracted this code?

Reply:
added